doc/timers.adoc

```diff
@@ -35,7 +35,7 @@ Each task gets a timer. The timer has three properties:
 - A _notification set._
 
 At periodic intervals (ticks), if the enable bit is set, the kernel checks the
-deadline to see if it is in the future. If not (it is `<=` the current time),
+deadline to see if it is in the future. If not (it is \<=; the current time),
```

Comment:
Sorry to round trip this again, but, `\<=;` produces `&lt;=;` in the HTML output -- this needs to be just `\<=` without the semicolon.

Reply:
Whoops. Done

Reply:
You might want to keep the backticks here.
See this example: https://gist.github.com/fernandogmo/34032124e94c6b383277decd8631c687